network_test.go

```diff
@@ -0,0 +1,409 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2019

Reply:
Typically everything that we modify in 2019 should have "-2019" to extend the range or have 2019 included.  I just noticed this one since it was a new file.